src/webview.ts

```diff
@@ -0,0 +1,226 @@
+import * as vscode from 'vscode';
+import { CMakeCache } from './cache';
+import * as api from './api';
+import { CMakeDriver } from './drivers/driver';
+
+interface IOption {
```

Comment:
I will take a look at this next.  I just wanted to get initial feedback out so I have not reviewed this file yet.